### PR TITLE
Refactor message build/get functions and add 2025 board IDs

### PR DESCRIPTION
**Pull request by JasonBrave**

### CODE REVIEW

File: message_types.h [hunk 1]
```diff
@@ -4,85 +4,102 @@
...
+    PRIO_LOW = 0x3,
+};
 
-#define MSG_SENSOR_TEMP           0x540
-#define MSG_SENSOR_ALTITUDE       0x560
-#define MSG_SENSOR_ACC            0x580
-#define MSG_SENSOR_ACC2           0x5A0
-#define MSG_SENSOR_GYRO           0x5E0
+// Message Types
+enum MESSAGE_TYPES {
```
Comment (celery6) [inline message_types.h:18]: image

File: message_types.h [hunk 2]
```diff
@@ -4,85 +4,102 @@
...
-#define MSG_DEBUG_PRINTF          0x1E0
-#define MSG_DEBUG_RADIO_CMD       0x200
-#define MSG_ACT_ANALOG_CMD        0x220
-
-
-#define MSG_ALT_ARM_STATUS        0x440
-#define MSG_ACTUATOR_STATUS       0x460
-#define MSG_GENERAL_BOARD_STATUS  0x520
+// Message Priority
+enum MESSAGE_PRIO {
```
Comment (celery6) [inline message_types.h:10]: naming convention same as below

File: message_types.h [hunk 3]
```diff
@@ -4,85 +4,102 @@
...
-#define MSG_STATE_EST_CALIB       0x620
-#define MSG_SENSOR_MAG            0x640
-#define MSG_SENSOR_ANALOG         0x6A0
-#define MSG_GPS_TIMESTAMP         0x6C0
-#define MSG_GPS_LATITUDE          0x6E0
-#define MSG_GPS_LONGITUDE         0x700
-#define MSG_GPS_ALTITUDE          0x720
-#define MSG_GPS_INFO              0x740
+// Board Type IDs
+enum BOARD_TYPE_ID {
```
Comment (celery6) [inline message_types.h:50]: naming convemtion

File: message/msg_common.h [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+#warning BOARD_TYPE_UNIQUE_ID not defined, please set that up in project
+#define BOARD_TYPE_UNIQUE_ID 0
+#endif
+
+#ifndef BOARD_INST_UNIQUE_ID
+#warning BOARD_INST_UNIQUE_ID not defined, please set that up in project
+#define BOARD_INST_UNIQUE_ID 0
+#endif
+
+#define SID(prio, msg)                                                                             \
```
Comment (celery6) [inline message/msg_common.h:22]: does this need to be a macro

**Review by celery6 — CHANGES_REQUESTED** [review]

File: message/msg_common.h (hunk 4 above)
Comment (JasonBrave) [reply message/msg_common.h:22]: This macro is just to replace previously copy and pasted logic OR

**Review by celery6 — APPROVED** [review]
lgtm

**Pull request merged**